spec/classes/puppet_agent_config_spec.rb

```diff
@@ -13,66 +13,68 @@
         })
       end
 
-      context 'with default parameters' do
-        let :pre_condition do
-          'include ::puppet'
-        end
+      unless facts[:osfamily] == 'windows'
```

Comment:
Can also be written as ```context 'with default parameters', :if => (facts[:osfamily] != 'windows') do```. I think it's cleaner because you end up with fewer indents. Also results in a smaller diff :)